src/Service/LinuxTracingHandler.cpp

```diff
@@ -60,6 +62,13 @@ void LinuxTracingHandler::SetupIntrospection() {
       });
 }
 
+void LinuxTracingHandler::SetupOrbitApi(const CaptureOptions& capture_options) {
+  auto result = orbit_api::InitializeInTracee(capture_options);
+  if (result.has_error()) {
+    LOG("Warning: Could not initialize Orbit Api: %s", result.error().message());
```

Comment:
I think this could just be an `ERROR`, we don't have a distinction between `ERROR` and less severe warnings (and this is actually quite severe).

Reply:
Yeah, switched to ERROR, thanks